nrfx/drivers/src/prs/nrfx_prs.h

```diff
@@ -142,6 +142,27 @@ extern "C" {
     #define NRFX_PRS_BOX_8_ADDR     NRF_UARTE120
     // COMP, LPCOMP
     #define NRFX_PRS_BOX_9_ADDR     NRF_COMP
+#elif defined(NRF54H20_ENGB_XXAA)
```

Comment:
Isn't the below set the same for NRF54H20_XXAA, NRF54H20_ENGA_XXAA, and NRF54H20_ENGB_XXAA? Perhaps this could be simplified on the next occasion?

Reply:
sure, I will add it to the next nrfx release scope